### ISSUE TITLE
Collection field spins if you don't select a collection in the field set

### ISSUE THREAD

**Issue by thestepafter** [statamic/v2-hub#1389]
### Expected behavior
I expect to see a list of collection entries by collection in the drop down.

### Actual behavior
The field never loads and just shows the spinner.

### Steps to reproduce
1. Add Collection field to field set but don't select a collection.
2. Assign the fieldset to a taxonomy.
3. View the taxonomy and see the spinning.

### Server Details

**Operating System**:
OSX

**Web Server:**
nginx

**PHP Version:**
7.0

**Statamic Version:**
2.5.8

**Updated from an older Statamic or fresh install:**
Fresh, like my Grandmother.

**List of installed add-ons:**
None.

### Logs
![screen shot 2017-04-13 at 6 30 41 pm](https://cloud.githubusercontent.com/assets/1793867/25027113/56c39966-2078-11e7-9516-de4ae0769ad5.png)
![screen shot 2017-04-13 at 6 31 43 pm](https://cloud.githubusercontent.com/assets/1793867/25027115/58a1e1ac-2078-11e7-983e-ad96c6b03e5e.png)

**Comment by jasonvarga** [comment]
You aren't selecting a collection, in a collection field?

Are you trying to show _all_ entries?

**Comment by thestepafter** [comment]
Yes, that is correct, ideally, I would see all collection entries for all collections since I didn't select a collection in the fieldset definition. If this isn't possible and a collection needs to be selected in the fieldset definition I understand but it should be required so that the spinning doesn't happen.
